hex/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "fast-hex"
+version = "0.1.0"
+authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>"]
+license = "CC0-1.0"
+description = "Fast hex encoding and decoding"
+homepage = "https://github.com/rust-bitcoin/hex/"
+repository = "https://github.com/rust-bitcoin/"
+documentation = "https://docs.rs/fast-hex/"
+keywords = ["encoding", "hex", "hexadecimal"]
+readme = "README.md"
+edition = "2018"
+
+[features]
+default = ["std"]
+std = ["alloc"]
+# If you disable std, you can still use Read/Write via the "io" feature.
+io = ["core2"]
+# If you disable std, you can still use Vec/String via the "alloc" feature.
+alloc = []
+
+[package.metadata.docs.rs]
+features = ["std", "alloc", "io"]
+rustdoc-args = ["--cfg", "docsrs"]
+
+[dependencies]
+# Only enable this if you explicitly do not want to use an allocator, otherwise enable "alloc".
```

Comment:
This comment looks out of place.

Reply:
Ah thanks, too many PRs and too much rebasing yesterday.